frontend/src/firebase/firebaseInteractor.ts

```diff
@@ -133,12 +151,26 @@ export default class FirebaseInteractor {
           userData.currentInterventionOrAssessment || "oiBN8aE5tqEFK2gXJUpl",
         daysActive:
           userData.daysActive === undefined ? [] : userData.daysActive,
+        profileIcon: userData.profileIcon ?? allProfileIcons[0],
       };
     } else {
       throw new Error("Invalid user");
     }
   }
 
+  /**
+   * Update user's profile icon; creates the field for profile icon if not already exists
+   * @param {String} url: the profile icon url
+   */
+  async updateUserProfileIcon(url: string) {
+    await this.db.collection("users").doc(this.auth.currentUser?.uid).set(
```

Comment:
I think we should use `update` here so we don't need the second parameter. If we use `update` it has the same behavior, but is a bit clearer in my opinion.

Reply:
ya for update im thinking is that for old users which don't have profile icon right, then u technically are updating it but if u dont have a field, wouldn't that just give u an error?